content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,521 @@
+---
+title: "Reference Application using RDT for COVID-19 Testing"
+linkTitle: "COVID-19 Testing"
+weight: 
+description: >
+ A customizable CHT application with support for 3rd party RDT integration   
+relatedContent: >
+  apps/reference/forms/app/#android-app-launcher
+  apps/reference/forms/app/#cht-xform-widgets
+  apps/reference/forms/app/#parse-timestamp-to-date
+---
+
+Medic has worked with [FIND](https://www.finddx.org) to provide a reference application using rapid diagnostic tests (RDT) for COVID-19 testing with the CHT.  By leveraging the expertise of third-party applications, like [Dimagi's RD-Toolkit](https://github.com/dimagi/rd-toolkit/), you can customize this application in your current or future deployment of the CHT. 
+
+You can find the code for this application in the [CHT Core repository on GitHub](https://github.com/medic/cht-core/tree/master/config/covid-19).
+
+## Problem Being Addressed
+
+The original call for proposals best describes why Medic created this app: 
+
+> FIND is working towards supporting countries in implementing an effective test-trace-isolate response using digital tools. To this end, FIND is looking to partner with leading digital solution providers to accelerate the development and deployment of a set of minimum functionalities for the collection of COVID RDT data and supporting incorporation of these functionalities into existing digital tools for use in low- and middle-income country settings.
+
+## Solution Overview
+
+This reference app provides a base layer of functionality that you can easily customize to meet the needs of your health program.  Specifically, this application has examples of:
+
+ * Provisioning COVID-19 RDT tests 
+ * Capturing COVID-19 RDT results
+ * Storing all RDT data, including pictures, in the CHT
+ * Best practices for RDT workflows in the CHT
+ 
+ Your instance of CHT needs to be on v3.13+ which has features developed to support this reference application, including:
```

Comment:
Is cht-android 0.10.0 implied, or should it be stated explicitly?

Reply:
Having RD-Toolkit is a requirement for this particular app too, where can that be explained?

Reply:
Good questions both! Answered in 78e19ce.